Azrux/Athenas-front
# [feat] - Create Carousel common component

## Description
Create Carousel component 

## Screenshots
![image](https://github.com/Azrux/Athenas-front/assets/89431511/814565e6-35e9-4979-893c-31df530bbda5)

## How to test?
run `npm run dev`
Go to: [http://localhost:3000/test](http://localhost:3000/test)
Check the design of the carousel and its functionality
